packages/frontend-2/components/settings/workspaces/Security.vue

```diff
@@ -69,17 +77,14 @@
           </div>
         </div>
       </section>
-      <hr class="my-6 md:my-8 border-outline-2" />
-      <section class="flex flex-col space-y-3">
-        <SettingsSectionHeader title="Domain features" subheading class="mb-3" />
+      <section class="flex flex-col space-y-3 mt-8">
         <div class="flex flex-col space-y-8">
           <div class="flex items-center">
             <div class="flex-1 flex-col pr-6 gap-y-1">
               <p class="text-body-xs font-medium text-foreground">Domain protection</p>
-              <p class="text-body-xs text-foreground-2 leading-5 max-w-md">
-                Admins won't be able to add users as members (or admins) to a workspace
-                unless the one of the users email matches one of the workspace's
-                verified email domains.
+              <p class="text-body-2xs text-foreground-2 leading-5 max-w-md">
+                Members won’t be able to add users as members (or admins) to a workspace
```

Comment:
Members already can't invite new people to the workspace, maybe stick with `Admin` in this wording